Tools/ci-run.sh

```diff
@@ -187,6 +187,6 @@ python runtests.py \
 
 EXIT_CODE=$?
 
-ccache 2>/dev/null -s -v
+ccache 2>/dev/null -v -v
```

Comment:
What's wrong with `-s`?

Reply:
Nothing is wrong
1 -s is already used in the post cache step
2 I wanted to see more verbosity

Reply:
But I guess it does nothing without -s

Reply:
I used `-s` before and after the test runs to be able to compare the cache usage.

Reply:
I see, It's kind of burried in the logs

Reply:
> I used `-s` before and after the test runs to be able to compare the cache usage.

isn't it the "cache hit rate" from the `-s` output?
So is there any need to do `-s` before the running tests?